sdk/python/feast/infra/aws.py

```diff
@@ -1,23 +1,165 @@
+import base64
 import os
 import uuid
 from datetime import datetime
 from pathlib import Path
 from tempfile import TemporaryFile
+from typing import Sequence, Union
 from urllib.parse import urlparse
 
 from colorama import Fore, Style
 
 import feast
-from feast.constants import AWS_LAMBDA_FEATURE_SERVER_IMAGE
+from feast import Entity, FeatureTable, FeatureView, __version__
```

Comment:
nit, in internal code we should be using fully qualified import statements. i.e. 
```
from feast.entity import Entity
```

instead of 
```
from feast import Entity
```